src/main/java/digital/slovensko/autogram/server/AssetsEndpoint.java

```diff
@@ -0,0 +1,65 @@
+package digital.slovensko.autogram.server;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import digital.slovensko.autogram.server.dto.ErrorResponse;
+import digital.slovensko.autogram.server.errors.InvalidUrlParamException;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+
+public class AssetsEndpoint implements HttpHandler {
+    private static final List<File> assets = new ArrayList<>();
+
+    static  {
+        var location = AssetsEndpoint.class.getProtectionDomain().getCodeSource().getLocation().getPath();
+        var path = Path.of(location,"digital", "slovensko", "autogram", "server", "assets");
+
+        Collections.addAll(assets, requireNonNull(path.toFile().listFiles()));
+    }
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        var uriPath = exchange.getRequestURI().getPath().substring(7); // remove /assets
```

Comment:
Ak chceme odstrániť /assets, tak to spravme regexom a určite nie takto nejakým magickým číslom.

Reply:
áno, toto riešenie je určite rýchlejšie. Avšak rozdiel je nepovšimnuteľný, ak sa to púšťa iba raz a nie 10K krát. Ak by sme sa všeobecne hrali na najefektívnejší kód, písali by sme všetko v assembleri, ale to by bolo strašne komplikované a nikto by tomu kódu nerozumel.

Často sa pri takýchto projektoch treba rozhodnúť medzi super efektivitou a čitateľnostťou kódu. A veľa krát je to o tom, že to teoretické zlepšenie v praxi neznamená nič, keď sa tá funkcia vykonáva konštantne veľa krát.